examples/bzlmod/MODULE.bazel

```diff
@@ -10,3 +10,25 @@ local_path_override(
     module_name = "rules_python",
     path = "../..",
 )
+
+python = use_extension("@rules_python//python:extensions.bzl", "python")
+
+python.toolchain(
```

Comment:
Does every root module have to manually register a toolchain or is the host Python used if nothing else is specified?

Reply:
In the case of this example, I had to use it to fix the python version because the generated `requirements_lock.txt` file stamps the python version in a comment, and on CI the different platforms had different versions installed, so the diff_test was failing.